docs/codacy-api/examples/identifying-commits-without-coverage-data.md

```diff
@@ -0,0 +1,73 @@
+---
+description: Example of how to identify commits missing the coverage data required for calculating the coverage metrics of pull requests.
+---
+
+# Identifying commits without coverage data
+
+To calculate the supported coverage metrics for pull requests, Codacy requires that at least the following commits provide coverage data:
+
+-   The common ancestor commit of the pull request branch and the target branch
+-   The head commit of the pull request branch
+
+The following diagram highlights the commits that must have received coverage data for Codacy to display the coverage variation metric on a pull request:
+
+![Commits that must have received coverage data](../../coverage-reporter/images/coverage-pr-commits.png)
+
+However, different factors may prevent your setup from correctly reporting coverage data for the required commits.
+
+To check if Codacy has received the required coverage data to calculate the coverage metrics for a pull request, use the Codacy API endpoint [getPullRequestCoverageReports](https://api.codacy.com/api/api-docs#getpullrequestcoveragereports).
+
+## Example: Identifying which pull request commits are missing coverage data
+
+This example checks whether the open pull requests in a repository have received coverage data for their head and common ancestor commits.
+
+The example script:
+
+1.  Defines the [account API token](../api-tokens.md#account-api-tokens) used to authenticate on the Codacy API, the Git provider, the organization name, and the repository name passed as an argument to the script.
+1.  Calls the Codacy API endpoint [listRepositoryPullRequests](https://api.codacy.com/api/api-docs#listrepositorypullrequests) to retrieve the list of open pull requests on the repository.
+1.  Uses [jq](https://github.com/stedolan/jq) to select only the numbers that identify the pull requests on the Git provider.
+1.  For each pull request, outputs the pull request number and calls the Codacy API endpoint [getPullRequestCoverageReports](https://api.codacy.com/api/api-docs#getpullrequestcoveragereports) to obtain the information about the coverage data received for the head and common ancestor commits of the pull request.
+1.  Uses [jq](https://github.com/stedolan/jq) to select and output the commit SHA-1 and coverage status for the commits.
+
+```bash
+CODACY_API_TOKEN="<your account API token>"
+GIT_PROVIDER="<your Git provider>" # gh, ghe, gl, gle, bb, or bbe
```

Comment:
Would it make sense to explain the abbreviations?

Reply:
We do have a couple of API example pages that [follow a format that "handholds" the users a bit more](https://docs.codacy.com/codacy-api/examples/adding-people-to-codacy-programmatically/#adding-people-to-codacy-programmatically). However, both on the other examples and the [API reference](https://api.codacy.com/api/api-docs#getpullrequestcoveragereports) docs we don't have the explicit mapping but also never had reports of users not figuring this out. :thinking: 

For now, I'd prefer to keep the page a bit simpler but your feedback makes a lot of sense and if we ever understand that users have trouble understanding this we'll add the extra info across all examples for consistency. :+1: